Improve docs for empty value in cache compression config (#9649)
grafana/loki: docs/sources/configuration/_index.md

  # frontend.index-stats-results-cache
  [cache: <cache_config>]

>   # Use compression in cache. Supported values are: 'snappy' and ''. Default ''.

Reviewer: ```suggestion
  # Use compression in cache. The default is an empty value '', which disables compression.
```

>   # An empty value (i.e. '') disables compression.

Author: Done
